content/_index.md

```diff
@@ -3,25 +3,40 @@ title: Kubernetes Contributors
 description: Welcome to the Kubernetes community!
 ---
 
-This is the starting point for joining and contributing to the Kubernetes
-community---improving docs, improving code, giving talks, etc. To learn more
-about the project structure and organization, please refer to Project Governance
-information.
 
-## Communicating
+This site is the home of contributor documentation, and serves as a starting point for your journey as a Kubernetes contributor.
 
-The communication page lists communication channels like chat, issues, mailing
-lists, conferences, etc.
+This site is to serve as the hub for all things related to the Kubernetes Contributor experience. Who exactly is a contributor? We all are--Whether you’re writing docs, reviewing code, participating in the community and its many SIGs, everyone is welcome. We hope this site will be a pathway to success for our 35000+ Kubernetes contributors, providing current, up-to-date information on community events, contributor resources, Kubernetes SIGs, and more.
 
-## Introduction
+### Get Involved:
 
-We can put anything we want here, news, Guin with cats, etc.
+The first step is to check out the [contributor guide](/guide/). All contributors must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
 
-{{< youtube -GayuFobHBo >}}
+If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack] and pop over to the [#kubernetes-dev channel][kdev].
 
-## Code Walkthrough
+Remember: No question is too small.
 
-If you are interested in contributing code, it is recommended that you begin
-with this code walkthrough by Stefan Schimanski
+### Stay Involved:
 
-{{< youtube yqB_le-N6EE >}}
+There are many ways you can stay involved with the Kubernetes Contributor community. We’d love to have you join a [SIG](https://git.k8s.io/community/sig-list.md), contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide, or hear your thoughts at one of our biweekly [contributor experience SIG meetings](https://github.com/kubernetes/community/tree/master/sig-contributor-experience#meetings). We welcome you to meet our contributors by joining our [monthly one hour meeting](https://github.com/kubernetes/community/blob/master/mentoring/meet-our-contributors.md), or by watching our Office Hours livestream on [YouTube](https://www.youtube.com/c/kubernetescommunity/live)!
```

Comment:
`contributor guide` can link to `/guide/` 👍 

Office Hours itself no longer uses that URL (a new one is generated for each one). As it's more enduser focused it might actually be worth removing it from the list.

Reply:
Removed this from the list.